### PR TITLE
images saving to firebase and some css fix

### PR DESCRIPTION
**Pull request by xaolaw**
review

### CODE REVIEW

File: fapp/src/pages/CreateExpense/CreateExpense.jsx [hunk 1]
```diff
@@ -33,6 +34,18 @@ export default function CreateExpense() {
 
   const { currentUser } = useContext(AuthContext);
 
+  const saveFile = (fileToUpload,newId) =>{
```
Comment (karol-t-wilk) [inline fapp/src/pages/CreateExpense/CreateExpense.jsx]: You could make this an async function and use await and try/catch

File: fapp/src/pages/CreateExpense/CreateExpense.jsx [hunk 2]
```diff
@@ -33,6 +34,18 @@ export default function CreateExpense() {
...
+    const storage = getStorage();
```
Comment (karol-t-wilk) [inline fapp/src/pages/CreateExpense/CreateExpense.jsx]: Shouldn't you import fileStorage from backend/firebase here?

File: fapp/src/pages/CreateExpense/CreateExpense.jsx [hunk 3]
```diff
@@ -33,6 +34,18 @@ export default function CreateExpense() {
...
+    const storageRef = ref(storage, `clients/${currentUser.uid}/${newId}/${fileToUpload.name}`) 
+    uploadBytes(storageRef, fileToUpload)
+    .then(() => {
+    })
+    .catch((error) => {
+      // Handle error
+      alert("Error uploading file:", error);
+    });
```
Comment (karol-t-wilk) [inline fapp/src/pages/CreateExpense/CreateExpense.jsx]: Consider using await with try/catch here

**Review by karol-t-wilk — CHANGES_REQUESTED** [review]

File: fapp/src/pages/CreateExpense/CreateExpense.jsx (hunk 2 above)
Comment (xaolaw) [reply fapp/src/pages/CreateExpense/CreateExpense.jsx]: ok

**Review by karol-t-wilk — APPROVED** [review]

**Pull request merged**